src/legacy/ui/public/vislib/lib/dispatch.js

```diff
@@ -244,33 +244,9 @@ export function VislibLibDispatchProvider(Private, config) {
     addBrushEvent(svg) {
       if (!this.isBrushable()) return;
 
-      const self = this;
       const xScale = this.handler.categoryAxes[0].getScale();
-      const brush = this.createBrush(xScale, svg);
-
-      function simulateClickWithBrushEnabled(d, i) {
-        if (!validBrushClick(d3.event)) return;
-
-        if (isQuantitativeScale(xScale)) {
-          const bar = d3.select(this);
-          const startX = d3.mouse(svg.node());
-          const startXInv = xScale.invert(startX[0]);
-
-          // Reset the brush value
-          brush.extent([startXInv, startXInv]);
-
-          // Magic!
-          // Need to call brush on svg to see brush when brushing
-          // while on top of bars.
-          // Need to call brush on bar to allow the click event to be registered
-          svg.call(brush);
-          bar.call(brush);
-        } else {
-          self.emit('click', self.eventResponse(d, i));
-        }
-      }
```

Comment:
I'm not clear why this is being removed? (Or TBH what it did in the first place...)

Reply:
Ah, nvm, I see now @markov00's comments on the original PR. Sounds like it's just dead code found in the course of fixing this